zebra-network/src/peer_set/candidate_set.rs

```diff
@@ -414,6 +414,8 @@ where
     }
 
     /// Returns the address book for this `CandidateSet`.
```

Comment:
```suggestion
    /// Returns the address book for this `CandidateSet`.
    ///
    /// TODO: Consider converting the candidate set to a tower service that owns the
    ///       address book and handles periodic tasks in a `poll_ready` method before using
    ///       this method in production.
```

Reply:
I think this TODO is already in this file at least once, or in a nearby file